assets/src/application/ui/input/Button/IconButton/index.tsx

```diff
@@ -2,24 +2,19 @@ import React from 'react';
 import classNames from 'classnames';
 
 import { IconButton as IconButtonAdapter, IconButtonProps as IconButtonAdapterProps } from '@infraUI/inputs';
-import { IconButtonProps } from './types';
-import { withLabel, withTooltip } from '../../display';
+import { IconButtonProps } from '../types';
+import { withLabel, withTooltip } from '../../../display';
+
+import './style.scss';
 
 type BtnType = React.ComponentType<IconButtonProps>;
 
 const IconButton = React.forwardRef<typeof IconButtonAdapter, IconButtonAdapterProps>(
-	({ className, icon, onClick, ...props }, ref) => {
-		const newClassName = classNames('esprs-button', className);
+	({ icon, onClick, ...props }, ref) => {
+		const className = classNames('ee-button ee-icon-button', props.className);
```

Comment:
might cause less conflicts (although potentially less DRY) to not apply `ee-button` class to icon buttons
i think before I was trying to do too much with the espresso button component

if there is a lot of duplication for the styles but we want to keep things DRY, then maybe we can add an `ee-btn-base` class that applies the common styles for both kinds of buttons